pygaggle/data/trec_covid.py

```diff
@@ -110,7 +110,7 @@ def to_relevance_examples(self,
                      for passage in passages][0])
             except ValueError as e:
                 logging.error(e)
-                logging.warning(f'Skipping passages')
+                logging.warning('Skipping passages')
                 continue
```

Comment:
Shouldn't we stop the execution instead of just skipping the error?

Reply:
I agree we should stop, let's check with @justinborromeo and address this in a separate PR.